app/models/ThangType.coffee

```diff
@@ -392,6 +392,12 @@ module.exports = class ThangType extends CocoModel
     props = _.without props, 'canCast', 'spellNames', 'spells'
     for stat, modifiers of itemConfig.stats ? {}
       stats[stat] = @formatStatDisplay stat, modifiers
+      if @heroStatModifier? and stats['maxHealth']? and !isNaN(stats['maxHealth'].display)
+        healthModifier = @heroStatModifier.health.absolute
+        totalHealth = Math.round(healthModifier * stats['maxHealth'].display)
+        modifiedHealth = Math.round(totalHealth - stats['maxHealth'].display)
+        stats['maxHealth'].display +=  "  (#{modifiedHealth})"
```

Comment:
I think we need a `+` in front of this if it's not negative, or it's kind of confusing.